hbshelpers/getDefaultUniversalLimit.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generates a default result limit for each vertical on the universal page. The
+ * default is taken from the vertical's `universalLimit`, specified in the related
+ * page's VTC. If there is no such `universalLimit`, the back-end default of 10 will
+ * be used.
+ * 
+ * @param {Object} pageConfigs - The configurations for each page.
+ * @returns {Object} The default search.universalLimit for the page.
+ */
+ module.exports = function getDefaultUniversalLimit(pageConfigs) {
+  const universalLimit = Object.entries(pageConfigs)
+    .filter(([key, _]) => key != 'index')
+    .reduce((limit, [_, config]) => {
+      const verticalKey = config.verticalKey;
+      const hasUniversalLimit = 
```

Comment:
the following code might be slightly simpler if this is just `const universalLimit = ...` but also feels like a personal preference thing

Reply:
So I had something like that initially.

```
const universalLimit = blah?.blah?.universalLimit
```

But somehow that syntax routinely caused our Percy snapshot generator to crash.